js/config/app.js

```diff
@@ -8,9 +8,8 @@ define(function () {
   };
   appConfig.cacheSources = false;
   appConfig.pollInterval = 60000;
-  appConfig.useBundled3dPartyLibs = false;
 	appConfig.cohortComparisonResultsEnabled = false;
-	appConfig.userAuthenticationEnabled = false;
+	appConfig.userAuthenticationEnabled = true;
```

Comment:
We shouldn't switch this default value from false to true, it's an incompatible change.